sdk/tables/azure-data-tables/samples/async_samples/sample_conditional_update_async.py

```diff
@@ -97,7 +96,7 @@ async def conditional_update_basic(self):
             except ResourceModifiedError:
                 print("This entity has been altered and may no longer be in the expected state.")
             entity3 = await table_client.get_entity(
-                partition_key=self.entity1["PartitionKey"], row_key=self.entity1["RowKey"]
+                partition_key=str(self.entity1["PartitionKey"]), row_key=str(self.entity1["RowKey"])
```

Comment:
We should not need `str()` here - it looks a bit redundant.

Reply:
Updated